bin/integration_test.sh

```diff
@@ -8,7 +8,15 @@
 # S3 bucket, invoking the submitter with an event, and then checking
 # the dest S3 bucket to see if the bits are there.
 
-set -e
+set -euo pipefail
+
+newtest() {
+    echo ""
+    echo "========================================================================"
+    echo ">>> $1"
+    echo "========================================================================"
+    echo ""
+}
```

Comment:
This makes it easier to see individual tests and follow the ridiculous integration test output.